examples/Remarketing/AddExpressionRuleUserList.cs

```diff
@@ -0,0 +1,158 @@
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Linq;
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V4.Common;
+using Google.Ads.GoogleAds.V4.Errors;
+using Google.Ads.GoogleAds.V4.Resources;
+using Google.Ads.GoogleAds.V4.Services;
+using static Google.Ads.GoogleAds.V4.Enums.UserListMembershipStatusEnum.Types;
+using static Google.Ads.GoogleAds.V4.Enums.UserListPrepopulationStatusEnum.Types;
+using static Google.Ads.GoogleAds.V4.Enums.UserListStringRuleItemOperatorEnum.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V4
+{
+    /// <summary>
+    /// This code example creates a rule-based user list defined by an expression rule for users who
+    /// have visited two different sections of a website.
+    /// </summary>
+    public class AddExpressionRuleUserList : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            AddExpressionRuleUserList codeExample = new AddExpressionRuleUserList();
+            Console.WriteLine(codeExample.Description);
+
+            // The Google Ads customer ID for which the call is made.
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            codeExample.Run(new GoogleAdsClient(), customerId);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description =>
+            "This code example creates a rule-based user list defined by an " +
+            "expression rule for users who have visited two different sections of a " +
+            "website.";
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The Google Ads customer ID for the conversion action is
+        /// added.</param>
+        public void Run(GoogleAdsClient client, long customerId)
+        {
+            // Gets the UserListService.
+            UserListServiceClient userListServiceClient =
+                client.GetService(Services.V4.UserListService);
+
+            // Creates the user targeting rules for each URL.
+            UserListRuleItemInfo rule1 = BuildVisitedSiteRuleInfo("example.com/section1");
+            UserListRuleItemInfo rule2 = BuildVisitedSiteRuleInfo("example.com/section2");
+
+            // Creates the containers for the two new rules.
```

Comment:
Missing [this comment](https://github.com/googleads/google-ads-java/blob/e00446021a94d0eb2da7e7ef088de60b94e488e9/google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/AddExpressionRuleUserList.java#L135), which I think it very important context.

Reply:
Agreed, missed it completely; has been added now.